### PR TITLE
add health check endpoint

### PR DESCRIPTION
**Pull request by kevinmekari**

### CODE REVIEW

File: lib/jurnal_api/client/cores.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module JurnalApi
+  class Client
+    # Defines methods related to Core
+    module Cores
+
+      def health_check(params = {})
+        response = get('core/ping/health_check', params)
```
Comment (rslhdyt) [inline lib/jurnal_api/client/cores.rb:7]: @kevinmekari could you please add spec for this endpoint?

**Review by rslhdyt — APPROVED** [review]

**Comment by rslhdyt** [comment]
@havus need your review

File: lib/jurnal_api/client/cores.rb (hunk 1 above)
Comment (kevinmekari) [reply lib/jurnal_api/client/cores.rb]: done mas @rslhdyt

**Comment by havus** [comment]
sorry for the late response, approved mas LGTM @kevinmekari

**Pull request merged**